layouts/partials/header.hbs

```diff
@@ -26,10 +26,8 @@
     <div class="switchers">
       <button class="dark-theme-switcher" type="button" title="{{site.toggleText}}" aria-label="Toggle dark/light mode"></button>
       <button class="lang-picker-toggler" type="button" aria-controls="lang-picker" aria-expanded="false">
-        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" width="25" height="25" aria-hidden="true" focusable="false" fill="currentColor">
-          <path d="M217.982 201.586h-64.499c-5.537 0-10.026 4.489-10.026 10.026s4.489 10.026 10.026 10.026h53.547c-4.72 25.263-26.935 44.446-53.547 44.446-30.037 0-54.473-24.436-54.473-54.473s24.436-54.473 54.473-54.473c14.55 0 28.229 5.667 38.518 15.955 3.916 3.916 10.264 3.916 14.178 0 3.916-3.916 3.916-10.264 0-14.178-14.077-14.077-32.791-21.829-52.697-21.829-41.094 0-74.525 33.431-74.525 74.525 0 41.094 33.431 74.525 74.525 74.525s74.525-33.431 74.525-74.525c.001-5.536-4.488-10.025-10.025-10.025z"/>
-          <path d="M470.331 92.24H269.728l-26.935-81.355a10.025 10.025 0 00-9.518-6.875H41.669C18.693 4.01 0 22.703 0 45.679v332.412c0 22.976 18.693 41.669 41.669 41.669h203.145l27.073 81.369a10.026 10.026 0 009.513 6.861h188.932c22.976 0 41.669-18.693 41.669-41.669V133.909c-.001-22.976-18.694-41.669-41.67-41.669zM41.669 399.708c-11.919 0-21.616-9.697-21.616-21.616V45.679c0-11.919 9.697-21.616 21.616-21.616h184.364l70.691 213.516a.366.366 0 00.015.043l53.664 162.086H41.669zm295.78-116.433c.805 1.11 10.824 14.877 26.355 34.066-4.377 5.756-9.015 11.474-13.91 17.036l-29.712-89.74h87.441c-6.196 13.031-16.938 33.813-31.692 55.736-13.553-16.921-22.069-28.622-22.249-28.87-3.251-4.482-9.519-5.481-14.002-2.23-4.482 3.25-5.48 9.518-2.231 14.002zM265.946 419.76h75.162l-55.503 59.084-19.659-59.084zm226.002 46.561c0 11.919-9.697 21.616-21.616 21.616H304.575l67.015-71.339-.004-.003c.293-.312.571-.64.823-.991a10.025 10.025 0 001.39-9.022l-16.688-50.402c7.073-7.406 13.68-15.143 19.805-22.965 13.299 15.772 29.037 33.446 45.778 50.187 1.957 1.957 4.524 2.937 7.089 2.937s5.132-.979 7.089-2.937c3.916-3.916 3.916-10.264 0-14.178-17.461-17.461-34.013-36.244-47.687-52.632 21.251-30.503 35.033-59.504 40.535-71.954h21.454c5.537 0 10.026-4.489 10.026-10.026s-4.489-10.026-10.026-10.026h-66.173v-18.047c0-5.537-4.489-10.026-10.026-10.026s-10.026 4.489-10.026 10.026v18.046h-51.406l-37.178-112.292H470.33c11.919 0 21.616 9.697 21.616 21.616v332.412z"/>
-        </svg>
+        <img class="lang-picker-icon light-icon" src="/static/images/lang-picker-icon-light.svg" alt="Language">
```

Comment:
you shouldn't use `img` tags. Instead inline them like before

Reply:
Then add a class and change the `fill` value only conditionally. That way you can use one icon.

Reply:
Example optimized SVG which you can then add a class and toggle the fill rule accordingly instead of having two icons:

```xml
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" width="3214.947" height="3725.333" viewBox="0 0 3214.947 3725.333">
  <path fill="none" stroke="#fff" stroke-width="53.332" d="M1606.093 478.88 362.44 39.947v2706.786l1243.653-402.36V478.88"/>
  <path fill="#333" d="M1576.24 477.52 2867.507 38.6v2706.787l-1291.267-402.36V477.52M39.947 3220.573l1536.293-512.066V477.227L39.947 989.293v2231.28"/>
  <path fill="#fff" d="m2276.28 3254.28 217.733 358.213 114.814-332.76ZM562.32 1246.333c-8.16-8.013 10.627 65.48 36.773 91.92 46.36 46.774 82.574 52.8 101.854 53.574 42.666 1.706 95.32-10.627 126.586-23.734 30.254-12.906 83.267-39.973 103.334-79.453m406.6 836.253c-12.107-4.4-262.587-108.16-298.094-125.16-29.053-13.973-100.293-44.093-133.813-57.773 94.413-145.573 154.013-255.427 161.947-272.16 14.68-30.613 114.613-226.16 116.946-238.2 2.267-12.2 5.107-57.267 2.907-67.973-2.2-10.92-38.84 10.066-88.587 26.933-49.826 16.8-144.52 78.387-181.093 86.107-36.707 7.653-154.013 52.093-214.04 72.013-60.027 19.92-173.573 54.573-220.28 67.187-46.773 12.613-87.6 13.613-113.76 21.546 0 0 3.48 36.64 10.427 47.627 6.866 10.987 31.613 37.92 60.386 45.44 28.774 7.573 76.4 4.533 98.094-.427 21.68-5.04 59.24-23.386 64.28-31.4 5.093-8.146-2.627-33.24 5.946-40.826 8.654-7.507 122.974-34.227 166.134-47.267 43.16-13.267 208.373-70.173 230.773-67.267-7.093 23.534-139.987 286.76-182.787 365.294-42.813 78.52-291.506 423.973-344.453 484.853-40.187 46.28-137.573 164.707-171.307 191.427 8.507 2.346 68.814-2.827 79.8-9.627 68.467-42.173 182.507-184.133 219.227-227.373 109.147-128 205.04-262.454 281.08-377.84h.08c14.813 6.173 134.587 103.76 165.84 125.386 31.253 21.614 154.587 90.427 181.307 101.84 26.72 11.56 129.413 58.907 133.733 42.88 4.32-16.16-18.573-110.64-30.693-115.24"/>
  <path fill="#fff" fill-rule="evenodd" d="M758.667 3429.333c24 14.667 46.666 26.667 72 38.667 50.666 25.333 108 52 162.666 72 74.667 28 149.334 50.667 224 68 41.334 9.333 86.667 17.333 130.667 24 4 0 122.667 14.667 146.667 14.667h120c46.666-4 90.666-6.667 137.333-13.334 37.333-5.333 78.667-12 118.667-21.333 29.333-6.667 60-13.333 89.333-22.667 28-8 60-18.666 90.667-29.333 20-6.667 41.333-16 62.666-24 17.334-8 38.667-17.333 58.667-25.333 24-10.667 52-25.334 78.667-38.667 21.333-10.667 45.333-24 68-37.333 17.333-9.334 57.333-40 78.666-40 24 0 40 21.333 40 40 0 38.666-52 50.666-76 68-25.333 17.333-56 30.666-82.666 45.333-53.334 28-108 52-160 72-68 25.333-142.667 49.333-209.334 65.333-25.333 5.334-50.666 12-76 16-13.333 2.667-152 24-190.666 24h-176c-46.667-4-96-9.333-142.667-16-41.333-6.666-85.333-14.666-126.667-24-32-6.666-66.666-16-97.333-25.333-53.333-14.667-105.333-33.333-156-53.333-92-34.667-188-80-278.667-140-16-10.667-17.333-21.334-17.333-33.334 0-20 14.667-38.666 38.667-38.666 21.333 0 64 30.666 72 34.666M1616 472v2240c-1.333 6.667-4 13.333-9.333 20-2.667 4-8 9.333-12 10.667C1561.333 2756 60 3260 40 3260c-16 0-30.667-10.667-38.667-28 0-1.333-1.333-2.667-1.333-5.333V985.333c2.667-6.666 4-16 9.333-21.333C20 949.333 38.667 946.667 50.667 942.667c22.666-8 1501.333-504 1522.666-504C1586.667 438.667 1616 448 1616 472zm-81.333 2206.667-1453.334 484v-2144l1453.334-484v2144"/>
  <path fill="#fff" fill-rule="evenodd" d="M2908 42.667V2740c-1.333 30.667-22.667 44-42.667 44-17.333 0-142.666-42.667-164-49.333-168-52-337.333-104-504-156-37.333-12-76-24-112-36-32-9.334-66.666-20-98.666-30.667-142.667-44-288-88-430.667-136-5.333-1.333-18.667-20-18.667-24V468c2.667-6.667 5.334-14.667 12-20 10.667-12 468-164 648-224 48-17.333 649.334-224 668-224 24 0 42.667 17.333 42.667 42.667zm-81.333 2646.666-1209.334-376V506.667L2826.667 96v2593.333"/>
  <path fill="#fff" d="m3214.947 3254.28-1615.614-514.947 6.76-2241.733 1608.854 512.067V3254.28"/>
  <path fill="#333" fill-rule="evenodd" d="m2305.307 1137.587 208.146 63.053 379.214 1366.693-213.8-64.866-76.814-280.574-441.84-133.92-95.013 228.56-213.867-64.88zm95.16 361.813-158.574 383.28 291.534 88.373z"/>
</svg>
```

I kept the line breaks but they can also be removed later if we want to.